### PR TITLE
Add Arch install instructions

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ make build
 sudo make install
 ```
 
+## Arch-based
```
Comment (CleanMachine1) [inline README.md:29]:
```suggestion
### Arch-based
```

**Comment by marchersimon** [comment]
Hmm, I actually thought about rewriting the installation part in the README completely. Maybe so:
# Installation

On Arch-based systems, you can install `tldr-cpp` with
```Bash
yay -S tldr-cpp-git
```
On other systems you have to install it manually.
```Bash
git clone https://github.com/marchersimon/tldr-cpp
cd tldr-cpp
make
sudo make install
```
To remove it agian, just run
```Bash
sudo rm /usr/bin/tldr
```

**Comment by CleanMachine1** [comment]
Sure I agree, I like making READMEs but I am not very good at design of them.

**Comment by marchersimon** [comment]
Me neither. But since I'm planning on expanding it more in the future, I try one heading per topic and not one per paragraph.

**Comment by marchersimon** [comment]
Do you think using `yay` for downloading the package is a good idea? On Manjaro for example it's not installed by default. Here, I'd use `pamac`. How would you do that on Arch? Can't you just access the AUR with `pacman`?

**Comment by CleanMachine1** [comment]
Most installs use yay, I prefer yay and its more the standard than anything.

If you really wanted the instructions could be

`https://aur.archlinux.org/tldr-cpp-git.git && cd tldr-cpp-git && makepkg -si`

**Comment by CleanMachine1** [comment]
But at that, isn't it easier to just run the standard commands.

Why not just write beneath it, `or your specific AUR package manager`

**Comment by marchersimon** [comment]
In this case I think it's best to just specify `yay`.

**Comment by marchersimon** [comment]
I've now done a major update to the install instructions, so I'll close this, but still thanks for your PR!

**Pull request closed without merging**